tests/src/Kernel/GroupMembershipConditionTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Drupal\Tests\og\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests the GroupMembership condition plugin.
+ *
+ * @group og
+ */
+class GroupMembershipConditionTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'entity_test',
+    'field',
+    'node',
+    'og',
+    'system',
+    'user',
+  ];
+
+  /**
+   * The condition plugin manager.
+   *
+   * @var \Drupal\Component\Plugin\PluginManagerInterface
+   */
+  protected $conditionManager;
+
+  /**
+   * The group type manager.
+   *
+   * @var \Drupal\og\GroupTypeManager
+   */
+  protected $groupTypeManager;
+
+  /**
+   * OG group node.
+   *
+   * @var \Drupal\node\Entity\node
+   */
+  protected $group;
+
+  /**
+   * OG role.
+   *
+   * @var \Drupal\og\Entity\OgRole
+   */
+  protected $role;
+
+  /**
+   * Member user.
+   *
+   * @var \Drupal\user\entity\User
+   */
+  protected $member;
+
+  /**
+   * Non-member user.
+   *
+   * @var \Drupal\user\entity\User
+   */
+  protected $nonMember;
+
+  /**
+   * Og Membership.
+   *
+   * @var \Drupal\og\Entity\OgMembership
+   */
+  protected $membership;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->conditionManager = $this->container->get('plugin.manager.condition');
+    $this->groupTypeManager = $this->container->get('og.group_type_manager');
+
+    // Add membership and config schema.
+    $this->installConfig(['og']);
+    $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('og_membership');
+    $this->installSchema('system', ['queue', 'sequences']);
```

Comment:
Special handling of system module schemas in \Drupal\KernelTests\KernelTestBase::installSchema has been deprecated in Drupal 8.7.x, remove any calls to this method that use invalid schema names. See https://www.drupal.org/node/3003360.

```suggestion
    $this->installSchema('system', ['sequences']);
```